### PR TITLE
added comand line input and some fixes

### PR DESCRIPTION
**Pull request by EvgenyKisel**

### CODE REVIEW

File: DEV-4/NondecreasingSequence/Inputer.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                foreach (var args in inputLine)
+                {
+                    Console.Write(args + " ");
+                }
+                int[] sequence = new int[inputLine.Length];
+                for (int i = 0; i < inputLine.Length; i++)
+                {
+                    sequence[i] = int.Parse(inputLine[i]);
+                }
+                return sequence;
```
Comment (IgorKheidorov) [inline DEV-4/NondecreasingSequence/Inputer.cs:23]: Use one return point

File: DEV-4/NondecreasingSequence/Inputer.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                foreach (var args in inputNumbers)
+                {
+                    Console.Write(args + " ");
+                }
+                int[] sequence = new int[inputNumbers.Length];
+                for (int i = 0; i < inputNumbers.Length; i++)
+                {
+                    sequence[i] = int.Parse(inputNumbers[i]);
+                }
+                return sequence;
```
Comment (IgorKheidorov) [inline DEV-4/NondecreasingSequence/Inputer.cs:38]: Use one return point

Comment (EvgenyKisel) [reply DEV-4/NondecreasingSequence/Inputer.cs:38]: But you said to leave it here and change it at dev-6 (

File: DEV-4/NondecreasingSequence/Inputer.cs (hunk 1 above)
Comment (EvgenyKisel) [reply DEV-4/NondecreasingSequence/Inputer.cs:23]: But you said to leave it here and change it at dev-6 (

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**